prefect_gcp/cloud_run.py

```diff
@@ -398,6 +407,30 @@ async def run(self, task_status: Optional[TaskStatus] = None):
             )
             return result
 
+    @sync_compatible
+    async def kill(self, identifier: str, grace_seconds: int = 30) -> None:
+        """
+        Kill a task running Cloud Run.
+
+        Args:
+            identifier: The Cloud Run Job name. This should match a
+                value yielded by CloudRunJob.run.
+        """
+        if grace_seconds != 30:
+            self.logger.warning(
+                f"Kill grace period of {grace_seconds}s requested, but GCP does not "
```

Comment:
This seems like a good way to handle these sorts of situations. @madkinsz how do you feel about setting a default value for `grace_seconds` and then logging a warning if the provided `grace_seconds` doesn't match as the standard way of doing things if we encounter this elsewhere?

Reply:
This looks like a copy of what I did in ECS :)

Reply:
:P yes taken from https://github.com/PrefectHQ/prefect-aws/pull/163/files#diff-31c73194d9fd094cabf83de9a5de6fb5be3f781ad540ed40f1cd4da589c49e4bR561